app/views/master_files/_player.html.erb

```diff
@@ -13,72 +13,22 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<% content_for :page_styles do %>
-  <%= stylesheet_link_tag 'mejs2_player' %>
-<% end %>
-<% content_for :page_scripts do %>
-  <%= javascript_include_tag "mejs2_player" %>
-<% end %>
+<% # Set MediaElement 4 query param variable
+is_mejs4 = params['mejs4'] === 'true' %>
```

Comment:
@cjcolvar I think I put this in here at some point, but was wondering your opinion...does it make sense to set this variable higher up in the application just once, so it can be accessed from any view/controller?